base/CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
-################################################################################
-# Copyright (C) 2014-2019 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
-#                                                                              #
-#              This software is distributed under the terms of the             #
-#              GNU Lesser General Public Licence (LGPL) version 3,             #
-#                  copied verbatim in the file "LICENSE"                       #
-################################################################################
-
-if(Geant4_FOUND AND Geant4VMC_FOUND)
-  add_subdirectory(sim/fastsim)
-endif()
+ ################################################################################
+ #    Copyright (C) 2014 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+ #                                                                              #
+ #              This software is distributed under the terms of the             #
+ #              GNU Lesser General Public Licence (LGPL) version 3,             #
+ #                  copied verbatim in the file "LICENSE"                       #
+ ################################################################################
+# Create a library called "libBase" which includes the source files given in
+# the array .
+# The extension is already found.  Any number of sources could be listed here.
```

Comment:
This comment adds more confusion than value, what array? what extension? That's why I removed it together with the CMake changes. It was also copied around to other targets a mullion times. Please don't re-add it...

Reply:
Sorry, it was added by accident, already removed it.